### ISSUE TITLE
Implement chicken animation

### ISSUE THREAD

**Issue by simonvonallmen-rethinc** [veganaut/veganaut#234]
At the end of a questionary ( issue #222 ) the user gets presented with a "thank you" screen. One part of this screen is an animation to display the message in a playful way. 

**implement the animation to the screen using the delivered file. Note that there will be several different animation available and we need a solution that randomly picks one of them to display.**

Attached you find the .zip file with the dedicated files for one animation.  It only runs on a server (not directly on localhost) therefor you need to startup a local server to test the animation ([animation demo](http://veganaut.simonvonallmen.ch/)). 

**Dedicated Screen in Zeplin:** https://zpl.io/V0PY8gx

**Animation-01:** [veganaut-animation-01.zip](https://github.com/veganaut/veganaut/files/1551680/veganaut-animation-01.zip)

**Comment by raydirty** [comment]
@simonvonallmen-rethinc In the zip file is currently only one animation, right? I could include all animations and select "randomly" an element. I think that's the best way to do it.

**Comment by simonvonallmen-rethinc** [comment]
@raydirty yes it only contains one animation. I'll create additional animations the next days/weeks and deliver them as well.

**Comment by simonvonallmen-rethinc** [comment]
Hey @raydirty I've finally had the time to create the additional questionary animations: 
- [veganaut_questionary-animation_01.zip](https://github.com/veganaut/veganaut/files/2343373/veganaut_questionary-animation_01.zip)
- [veganaut_questionary-animation_02.zip](https://github.com/veganaut/veganaut/files/2343374/veganaut_questionary-animation_02.zip)
- [veganaut_questionary-animation_03.zip](https://github.com/veganaut/veganaut/files/2343375/veganaut_questionary-animation_03.zip)
- [veganaut_questionary-animation_04.zip](https://github.com/veganaut/veganaut/files/2343376/veganaut_questionary-animation_04.zip)

Let me know if you need anything else to complete this issue.
